scripts/apothecary/formulas/cairo.sh

```diff
@@ -30,40 +30,81 @@ GIT_TAG=$VER
 # download the source code and unpack it into LIB_NAME
 function download() {
 	curl -LO http://cairographics.org/releases/cairo-$VER.tar.xz
-	tar -xf cairo-$VER.tar.xz
-	mv cairo-$VER cairo
-	rm cairo-$VER.tar.xz
-
+	if [ "$TYPE" == "vs" ] ; then
+		7z e cairo-$VER.tar.xz
```

Comment:
On my system, using ofMingGW, 7z command line tool is not available by default.  I was able to get it to compile just fine without this custom vs section.  Is there a reason why we need a custom 7z section?